dags/test_dag.py

```diff
@@ -0,0 +1,21 @@
+from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+from airflow import DAG
+from datetime import datetime
+
+default_args = {
+    'owner': 'airflow',
+    'start_date': datetime(2023, 1, 1),
+}
+
+with DAG('test_k8s_dag', default_args=default_args, schedule_interval=None) as dag:
+
+    k8s_task = KubernetesPodOperator(
+        namespace='default',
+        image='python:3.8-slim',

```

Comment:
Feel free to use any Python image for your transformations, etc 

As mentioned in previous comments, the image 'ghcr.io/bcgov/nr-dap-ods-ora2pg:main' is available to use for basic, like-to-like replication too.